### PR TITLE
Clarify storage recommendations for monitoring stack

### PR DESCRIPTION
**Pull request by jboxman**
- https://bugzilla.redhat.com/show_bug.cgi?id=1645358

### CODE REVIEW

**Comment by sichvoge** [comment]
@jboxman the title should say "monitoring stack" not logging ;) I added some comments and we are still missing the updates to the storage recommendation guide. You want to have a quick call sometime next week and go through the changes for that or are you ok to go ahead and do a first draft?

**Comment by jboxman** [comment]
@sichvoge, I'm available for a call sometime next week to go over the desired changes in greater detail. Thanks.

**Comment by jboxman** [comment]
@sichvoge, I've updated this PR with the content from the Google Doc. PTAL. Thank you!

File: install_config/monitoring/configuring-openshift-cluster-monitoring.adoc
Comment (ahardin-rh) [inline install_config/monitoring/configuring-openshift-cluster-monitoring.adoc]: environments, it

File: scaling_performance/optimizing_storage.adoc
Comment (ahardin-rh) [inline scaling_performance/optimizing_storage.adoc]: s/has shown/shows

storage and, therefore, is not

File: scaling_performance/scaling_cluster_monitoring.adoc
Comment (ahardin-rh) [inline scaling_performance/scaling_cluster_monitoring.adoc]: you must set the `openshift_cluster_monitoring_operator_prometheus_storage_enabled=true` in your Ansible inventory file to `true`.

**Comment by ahardin-rh** [comment]
@jboxman Just a few suggestions from me. :+1:

**Comment by sichvoge** [comment]
Is there a staging environment for this so that I can view the end result?

File: release_notes/ocp_3_11_release_notes.adoc
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: You can delete the trailing ==== after the heading

File: release_notes/ocp_3_11_release_notes.adoc
Comment (ahardin-rh) [inline release_notes/ocp_3_11_release_notes.adoc]: You can add a link to the OCS product documentation

File: release_notes/ocp_3_11_release_notes.adoc
Comment (jboxman) [reply release_notes/ocp_3_11_release_notes.adoc]: Oops, thanks!

**Comment by jboxman** [comment]
@sichvoge, this PR is ready to merge if it looks good to you.

Thanks!

**Comment by sichvoge** [comment]
Still waiting for Tushar or Sudhir to do peer review for their sections. I have sent them an email and as soon as I am back from PTO, will poke them again :)

**Comment by jboxman** [comment]
@sichvoge, ping

**Comment by sichvoge** [comment]
@jboxman sorry buddy, i will poke them again.

**Comment by jboxman** [comment]
@sichvoge, I can split out the changes that specifically cover recommendations, and exclude the references to OCS by name. Would that potentially require less scrutiny and thus be easier to merge?

**Comment by sichvoge** [comment]
@jboxman let's go ahead and merge the changes. i don't think that there is anything crazy in it and most is related to metrics anyways. the one i am a bit concerned is the terminology around OCS being only in a few places vs other parts of the doc just using "GlusterFS". 

but if that's ok w/ you, i am fine w/ it as well.

File: scaling_performance/optimizing_storage.adoc
Comment (sichvoge) [inline scaling_performance/optimizing_storage.adoc]: @jboxman can we actually leave that footnote but change it to: 

> For metrics, it is an anti-pattern to use any shared storage and a single volume (RWX).

I think that actually does represent that no one should ever configure RWX.

**Comment by sichvoge** [comment]
@jboxman added another review point. please check before merging this.

**Review by sichvoge — CHANGES_REQUESTED** [review]
@jboxman added another review point. please check before merging this.

File: scaling_performance/optimizing_storage.adoc
Comment (jboxman) [reply scaling_performance/optimizing_storage.adoc]: Remove _By default, metrics deploys with one volume per Cassandra Replica._ this.

File: scaling_performance/optimizing_storage.adoc
Comment (jboxman) [inline scaling_performance/optimizing_storage.adoc]: Add back this: _footnoteref:[metrics-warning,For metrics, it is an anti-pattern to use any shared storage and a single volume (RWX)._

File: scaling_performance/optimizing_storage.adoc
Comment (jboxman) [inline scaling_performance/optimizing_storage.adoc]: Add all changes for this file to a new PR against 4.1.

**Comment by jboxman** [comment]
Created https://github.com/openshift/openshift-docs/pull/15067 for 4.1.

**Comment by sichvoge** [comment]
So we closed this PR and don't fix our docs for 3.11? cc/ @vikram-redhat

**Comment by vikram-redhat** [comment]
> So we closed this PR and don't fix our docs for 3.11? cc/ @vikram-redhat

Nope. Not a deliberate attempt to close, but a side effect of us moving to the new repo structure in 4.1.  @jboxman should already have a new PR open shortly.

File: scaling_performance/optimizing_storage.adoc
Comment (jboxman) [reply scaling_performance/optimizing_storage.adoc]: Ultimately adopted this in 4.x: _footnoteref:[metrics-warning,For metrics, using file storage with the ReadWriteMany (RWX) access mode is unreliable. If you use file storage, do not configure the RWX access mode on any PersistentVolumeClaims that are configured for use with metrics.]_

**Pull request closed without merging**
